### PR TITLE
Allow providing ec2-instance-count to start multiple runners

### LINKED ISSUE #8: Start multiple runners for processing a couple of jobs in parallel

### PR DESCRIPTION
**Pull request by dexbleeker**
Fixes #8 

This PR is based on the fork of @messense, specifically [this](https://github.com/messense/huaweicloud-github-runner/commit/2354ded37963c009a9c03067cfe99eacd862676d) commit.

It allows for providing `ec2-instance-count` as input variable and will start multiple runners if desired. By default 1 runner is started.

Note that this might not be the cleanest code or best implementation. I don't write JS generally and have not touched a GH action before. Especially `Array.from(JSON.parse(ec2InstanceIds))` feels wrong. 

All feedback is welcome however.

### CODE REVIEW

**Comment by mickmorse** [comment]
Works like a charm 

@machulav 👀

File: dist/index.js
Comment (mgaitan) [inline dist/index.js:1]: what is this 12mb file used for?

**Comment by wu-json** [comment]
Any plans to merge this?

**Comment by Dmitry1987** [comment]
MinCount/MaxCount are not used anywhere, shouldn't it be a specific count, why there's a min/max, kind of "if at least N will successfully be created" ?
a useful feature for me too, i wonder why won't be merged :(

**Comment by thimios** [comment]
+1 Very useful feature for us too.

**Pull request closed without merging**